### PR TITLE
SB-148,SB-150,SB-151 drop a pollution sensor on the map

### PR DESCRIPTION
**Pull request by Yqiyuan**
# Description

https://user-images.githubusercontent.com/12007072/232249956-f60856cd-3a57-4fe4-8a11-124be54b29e7.mp4

# Test Plan

Please explain how this change will be tested.

- [x] Unit test
- [x] Integration test (including unit tests with supertest)
- [x] Manual test (Please explain below for the test steps)
- [ ] Not Applicable

# Task Link

Please add related jira task links

### CODE REVIEW

File: client/src/components/Map/Map.tsx [hunk 1]
```diff
@@ -45,6 +50,9 @@ export type layerSelectionhandler = (category: string, value: string) => void
 
 // helper component for mao popup
 const Popup: FunctionComponent<IPopupProps> = ({ type, onclick }) => {
+  if (type === 'pollution') {
```
Comment (jennyzeng) [inline client/src/components/Map/Map.tsx:53]: nit: maybe create a enum for types

File: client/src/components/Map/Map.tsx [hunk 2]
```diff
@@ -333,6 +349,27 @@ const SemMap: FunctionComponent<ISemMapProps> = ({
         )
         renderAreasOnMap({ coordinatesSet, map })
         break
+      case 'Pollution':
+        elements = await request(
+          `/api/mapelements/firstresponder?type=pollution`
+        )
```
Comment (jennyzeng) [inline client/src/components/Map/Map.tsx:355]: maybe move this request call out of this file

File: client/src/components/Map/Map.tsx [hunk 3]
```diff
@@ -333,6 +349,27 @@ const SemMap: FunctionComponent<ISemMapProps> = ({
         )
         renderAreasOnMap({ coordinatesSet, map })
         break
+      case 'Pollution':
+        elements = await request(
```
Comment (jennyzeng) [inline client/src/components/Map/Map.tsx:353]: may also need to handle the error

File: server/src/processConfig.ts [hunk 4]
```diff
@@ -26,6 +28,9 @@ const getConfig = (): ENV => {
       ? String(process.env.DB_HOST)
       : String('mongodb://localhost:27017/incident-response'),
     PORT: process.env.PORT ? Number(process.env.PORT) : 3001,
+    PURPLE_AIR_API_KEY: process.env.PURPLE_AIR_API_KEY
+      ? String(process.env.PURPLE_AIR_API_KEY)
+      : String('B8CC3B3A-DB23-11ED-BD21-42010A800008'),
```
Comment (jennyzeng) [inline server/src/processConfig.ts:33]: you may need to ask the guy managing the ci to add the key in ci and cd

**Review by jennyzeng — APPROVED** [review]

File: client/src/components/Map/Map.tsx (hunk 1 above)
Comment (Yqiyuan) [reply client/src/components/Map/Map.tsx:53]: The types have two different forms and are referenced in too many places in the code. I will make it a tech debt.

File: client/src/components/Map/Map.tsx (hunk 2 above)
Comment (Yqiyuan) [reply client/src/components/Map/Map.tsx:355]: Will record it as a refactoring issue

File: server/src/processConfig.ts (hunk 4 above)
Comment (Yqiyuan) [reply server/src/processConfig.ts:33]: Good suggestion, I will change that later.

**Review by bettsongy — APPROVED** [review]

File: client/src/components/Map/Map.tsx (hunk 3 above)
Comment (Yqiyuan) [reply client/src/components/Map/Map.tsx:353]: Will handle it in the refactoring issue

**Pull request merged**